packages/trader/tsconfig.json

```diff
@@ -1,8 +1,10 @@
 {
   "extends": "@tsconfig/node14/tsconfig.json",
-  "include": ["src/**/*.ts"],
+  "include": ["test/**/*.ts", "hardhat.config.ts"],
```

Comment:
Note: this only includes non-production files (aka all ts files outside of src).